components/section/info/src/index.js

```diff
@@ -4,12 +4,16 @@ import PropTypes from 'prop-types'
 const baseClass = 'sui-SectionInfo'
 
 class SectionInfo extends Component {
+  displayTitle (title) {
+    return (title !== undefined && title !== '')
+  }
+
   render () {
     const { title, children } = this.props
 
     return (
       <section className={baseClass}>
-        <h3 className={`${baseClass}-title`}>{title}</h3>
+        { this.displayTitle(title) && <h3 className={`${baseClass}-title`}>{title}</h3> }
```

Comment:
why not `{ title && <h3 className={`${baseClass}-title`}>{title}</h3>}`

Reply:
You're ·"$%ing right! Thanks

Reply:
But if you want to avoid to render the element with an empty string, you could do: `{Boolean(title) && <h3 className={'${baseClass}-title'}>{title}</h3>}` in a single line.

Reply:
😅